src/flair/flair_correct.py

```diff
@@ -14,7 +14,7 @@
 import logging
 
 def parseargs(aligned_reads=''):
-    parser = argparse.ArgumentParser(description='flair-correct parse options')
+    parser = argparse.ArgumentParser()
```

Comment:
hve a real description.  This is only English for what it does.  Not command signature, etc.  Those get generated.   This will lets us move to generating doc.